GameServer/commands/gmcommands/adviceTeam.cs

```diff
@@ -29,15 +39,16 @@ public void OnCommand(GameClient client, string[] args)
 			foreach (GameClient playerClient in WorldMgr.GetAllClients())
 			{
 				if (playerClient.Player == null) continue;
-				if ((playerClient.Player.Realm == client.Player.Realm ||
-				     playerClient.Account.PrivLevel > 1) && !playerClient.Player.IsIgnoring(client.Player))
+				
+				if (playerClient.Player.Realm == client.Player.Realm && playerClient.Account.PrivLevel > 1)
 				{
-					var name = "Atlas";
+					var name = "OpenDAoC";
+
 					// Message: [ADVICE {0}] {1}: {2}
-					ChatUtil.SendAdviceMessage(playerClient, "Social.SendAdvice.Msg.Channel", getRealmString(client.Player.Realm), name, msg);
+					ChatUtil.SendTypeMessage((int)eMsg.Advice, client, "Social.SendAdvice.Msg.Channel", getRealmString(client.Player.Realm), name, msg);
```

Comment:
`client` -> `playerClient`. This is sending the message to the user X times (X being the amount of clients).